ruby/meter-health/meter_health.rb

```diff
@@ -0,0 +1,117 @@
+require 'rubygems'
+require 'bundler/setup'
+require 'rest_client'
+require 'oauth2'
+require 'nokogiri'
+require 'json'
+require 'csv'
+
+class MeterHealth
+
+  ORGANIZATIONS = "/api/v2.json"
+  METERS = "/api/v2/organizations/%d/meters.json"
+  SITE_BASE_URL = 'http://54.90.173.204:8080'
+  SITE_OAUTH_PATH = '/oauth/token'
+
+  def initialize
+    @credentials = retrieve_credentials
+    @token = generate_token
+  end
+
+  def create_report
+    if @token
+      request_information
+    else
+      puts "Verify yor credentials, we weren't able to retrieve a token"
+    end
+  end
+
+  private
+  
+  def retrieve_credentials
+    file = File.read('./config/credentials.json')
+    JSON.parse(file)
+  end
+
+  def generate_token
+    client = OAuth2::Client.new(@credentials["app_id"], @credentials["app_secret"],
+      site: SITE_BASE_URL, token_url: SITE_OAUTH_PATH, scope: @credentials["scope"])
+    token = client.password.get_token(@credentials["user_email"],
+      @credentials["user_password"],scope: @credentials["scope"])
+    token and token.token ? token.token : nil
+  end
+
+  def request_information
+    organizations = JSON.parse(retrieve_organizations)
+    if organizations && organizations["embedded"]["organizations"]
+      write_to_csv(organizations["embedded"]["organizations"])
+    else
+      puts "There are not organizations"
```

Comment:
You can replace the message with something like: `There are no organizations` or `No organizations were found`.